templates/docs/examples/layouts/application/structure.html

```diff
@@ -19,7 +19,9 @@
         <button class="js-menu-pin is-dense u-no-margin u-hide--small">Pin</button>
         <button class="js-menu-close is-dense u-no-margin u-hide--medium">Close</button>
       </p>
-      <code>l-navigation &gt;<br> l-navigation__drawer</code>
+      <div class="p-card">
```

Comment:
I'd like to avoid such workarounds. Putting `p-card` here is quite unusual.

I guess the easiest way to avoid would be to get rid of the black background on this element (it's just added there for the sake of example) and use some other (`$color-mid-dark`?) that would work with inline code.

Reply:
Fair point - I tried with `$color-mid-dark`, but it seemed too dark, so I went with `$color-mid-x-light`, let me know what you think.

Reply:
@bartaz @sowasred2012 So it turns out we do actually use inline code on a dark background inside vanilla itself. 
I don't think changing the aside background isa good solution, this makes it look as if that's the default aside colour we recommend - and it doesn't work well with the shadow.

Can we just take the classes out of the inline code and haveit just as text?

![image](https://user-images.githubusercontent.com/2741678/107782484-09398b00-6d41-11eb-94ec-a429b686b790.png)

Reply:
@lyubomir-popov we know that we use it, this is the screenshot that allowed us to notice that we have this issue.

Previously we used black, which also is probably not a color we 'recommend' there - so this is really just a placeholder color so we can show the box structure of the layout. We can try using fade-out red as in grid examples to give it more of debugging look.

But yes, because these `<code>` elements are causing us issues here, I'm fine with replacing them with text (of maybe `<h5>` or something.